### PR TITLE
Update vmtest, expand arm testing

### PR DESCRIPTION
**Pull request by hugelgupf**

### CODE REVIEW

File: integration/gotests/gotest_test.go [hunk 1]
```diff
@@ -75,9 +76,9 @@ func testPkgs(t *testing.T) []string {
 		"github.com/u-root/u-root/pkg/tss",
 		"github.com/u-root/u-root/pkg/syscallfilter",
 	}
-	if qemu.GuestArch() == qemu.ArchArm64 {
-		blocklist = append(
-			blocklist,
+	switch qemu.GuestArch() {
+	case qemu.ArchArm64:
+		blocklist = append(blocklist,
 			"github.com/u-root/u-root/pkg/strace",
```
Comment (rminnich) [inline integration/gotests/gotest_test.go:82]: This ought to work, I think?
syscall_linux_arm64.go is in pkg/strace?

**Review by rminnich — DISMISSED** [review]
I *think* strace works on aarch64 (arm64)?

File: integration/gotests/gotest_test.go (hunk 1 above)
Comment (hugelgupf) [reply integration/gotests/gotest_test.go]: Let's give that a try.

arm64 qemu seems to be twice as slow as amd64 and arm. Not sure why that is.

Comment (hugelgupf) [reply integration/gotests/gotest_test.go]: `../../pkg/strace/syscalls_test.go:23:28: undefined: unix.SYS_OPEN`

test itself doesn't compile. I'll leave it for another time.

**Pull request merged**